src/main/java/org/jabref/gui/maintable/OpenShortScienceAction.java

```diff
@@ -0,0 +1,76 @@
+package org.jabref.gui.maintable;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.List;
+import java.util.Optional;
+
+import javafx.beans.binding.BooleanExpression;
+
+import org.jabref.gui.DialogService;
+import org.jabref.gui.StateManager;
+import org.jabref.gui.actions.SimpleCommand;
+import org.jabref.gui.desktop.JabRefDesktop;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.StandardField;
+
+import org.apache.http.client.utils.URIBuilder;
+
+import static org.jabref.gui.actions.ActionHelper.isFieldSetForSelectedEntry;
+import static org.jabref.gui.actions.ActionHelper.needsEntriesSelected;
+
+public class OpenShortScienceAction extends SimpleCommand {
+    private static final String BASIC_SEARCH_URL = "https://www.shortscience.org/internalsearch";
+    private final DialogService dialogService;
+    private final StateManager stateManager;
+
+    public OpenShortScienceAction(DialogService dialogService, StateManager stateManager) {
+        this.dialogService = dialogService;
+        this.stateManager = stateManager;
+
+        BooleanExpression fieldIsSet = isFieldSetForSelectedEntry(StandardField.TITLE, stateManager);
+        this.executable.bind(needsEntriesSelected(1, stateManager).and(fieldIsSet));
+    }
+
+    @Override
+    public void execute() {
+        stateManager.getActiveDatabase().ifPresent(databaseContext -> {
+            final List<BibEntry> bibEntries = stateManager.getSelectedEntries();
+
+            if (bibEntries.size() != 1) {
+                dialogService.notify(Localization.lang("This operation requires exactly one item to be selected."));
+                return;
+            }
+            getShortScienceSearchURL(bibEntries.get(0)).ifPresent(url -> {
+                try {
+                    JabRefDesktop.openExternalViewer(databaseContext, url, StandardField.URL);
+                } catch (IOException ex) {
+                    dialogService.showErrorDialogAndWait(Localization.lang("Unable to open ShortScience."), ex);
+                }
+            });
+        });
+    }
+
+    /**
+     * Get a URL to the search results of ShortScience for the BibEntry's title
+     * @param entry The entry to search for. Expects TITLE to be set for successful return.
```

Comment:
Please add an empty line above.

Please use `BibEntry's title` in both lines (title vs. TITLE) - or use `StandardFields.TITLE` here.